roles/lib_openshift/library/oc_adm_csr.py

```diff
@@ -1478,11 +1478,23 @@ def action_needed(csr, action):
 
         return False
 
+    def get_csr_request(self, request):
+        '''base64 decode the request object and call openssl to determine the
+           subject and specifically the CN: from the request
+
+           Output:
+           (0, '...
+                Subject: O=system:nodes, CN=system:node:ip-172-31-54-54.ec2.internal
+                ...')
+        '''
+        import base64
```

Comment:
Generally speaking, I agree imports belong at the top.  Since these are generated I'd prefer to localize the import as to minimize the files that will receive the import.

Reply:
Generally speaking I agree that it should be.  This is a generated file and to minimize the impact I'd prefer to localize it.

This is a stop gap until the product accepts node CSRs.